docs/en/sql-reference/functions/functions-index.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
Changes in this file were actually made in [71846](https://github.com/ClickHouse/ClickHouse/pull/71846) on main repo but I am committing it here as this PR fails without it. Same for `engines/index.md`